src/connection_string.ts

```diff
@@ -335,6 +335,19 @@ export function parseOptions(
     allOptions.set(key, values);
   }
 
+  if (allOptions.has('tlsCertificateKeyFile') && !allOptions.has('tlsCertificateFile')) {
+    allOptions.set('tlsCertificateFile', allOptions.get('tlsCertificateKeyFile'));
+  }
+
+  if (allOptions.has('tls') || allOptions.has('ssl')) {
+    const tlsAndSslOpts = (allOptions.get('tls') || [])
+      .concat(allOptions.get('ssl') || [])
+      .map(getBoolean.bind(null, 'tls/ssl'));
```

Comment:
```suggestion
    const tlsAndSslOpts = [...(allOptions.get('tls') ?? []), ...(allOptions.get('ssl') ?? [])].map(
      (value: boolean) => getBoolean('tls/ssl', value)
    );
```

It's totally preference, but this seems to avoid the bind. 
The other syntax is just 🧅 no need to use it, .concat works fine too.